app/Http/Controllers/GameController.php

```diff
@@ -21,6 +21,10 @@ public function getLootInGame($gameId) {
         return Game::all()->where('id', '=', $gameId)->first()->loots;
     }
 
+    public function getStatusInGame($gameId) {
+        return Game::all()->where('id', '=', $gameId)->first()->status;
```

Comment:
Misschien is het ook handig om een `404` melding te geven met een json bericht als "Er is geen game gevonden."

Reply:
Dit hebben we tot nu toe ook nergens anders gedaan. Later kunnen we dat altijd nog toevoegen,.